src/core/manager.h

```diff
@@ -448,6 +448,8 @@ struct Manager {
         VarlinkServer *varlink_server;
         /* Only systemd-oomd should be using this to subscribe to changes in ManagedOOM settings */
         Varlink *managed_oom_varlink_request;
+        /* Used by user managers to send ManagedOOM settings changes to systemd-oomd. */
+        Varlink *managed_oom_varlink;
```

Comment:
maybe even rename these two fields to:

managed_oom_varlink_system and managed_oom_user? 

i.e. i'd really like to underline what these two things are for, and now that they are two this migh tmean that the existing object should be renamed to make things clearer.